streamer/src/nonblocking/quic.rs

```diff
@@ -501,7 +502,14 @@ async fn setup_connection(
                     ),
                     |(pubkey, stake, total_stake, max_stake, min_stake)| {
                         let peer_type = if stake > 0 {
-                            ConnectionPeerType::Staked(stake)
+                            // If it is a staked connection with ultra low stake ratio, treat it as unstaked.
+                            let min_ratio = 1_f64 / (MAX_STREAMS_PER_MS * 100) as f64;
```

Comment:
What does 100 represent in this calculation?

Reply:
Yes. MAX_STREAMS_PER_MS * 100 is the max streams we allowed within the throttle interval.